seqr/views/apis/report_api.py

```diff
@@ -561,9 +562,9 @@ def _fetch_airtable_records(record_type, fields=None, filter_formula=None, offse
 
     if response_json.get('offset'):
         return _fetch_airtable_records(
-            record_type, fields=fields, filter_formula=filter_formula, offset=response_json['offset'], records=records)
+            record_type, user=user, fields=fields, filter_formula=filter_formula, offset=response_json['offset'], records=records)
 
-    logger.info('Fetched {} {} records from airtable'.format(len(records), record_type))
+    logger.info('Fetched {} {} records from airtable'.format(len(records), record_type), extra={'user': user})
```

Comment:
It seems that logging the HTTP requests (URL, path, method, size, etc.) is more important than logging the number of records fetched.

Reply:
actually no, the rest of that would just be logging constants because of the way we do error handling, except for size which is more useful as record number than request size anyways. Leaving this as is